packages/fhir-converter/src/templates/cda/ValueSet/MedicationStatementStatus.hbs

```diff
@@ -30,31 +30,37 @@
   {{#with (toLower code) as |normalizedCode|}}
     {{#if normalizedCode}}
       {{#if (or (eq normalizedCode "aborted") (eq normalizedCode "discontinued"))}}
-        "stopped"
+        "entered-in-error"
       {{else if (eq normalizedCode "stopped")}}
-        "stopped"
+        "entered-in-error"
       {{else if (eq normalizedCode "failed")}}
-        "stopped"
-      {{else if (eq normalizedCode "in-progress")}}
-        "in-progress"
-      {{else if (eq normalizedCode "active")}}
-        "in-progress"
-      {{else if (eq normalizedCode "not-done")}}
-        "not-done"
+        "entered-in-error"
       {{else if (eq normalizedCode "entered-in-error")}}
         "entered-in-error"
       {{else if (eq normalizedCode "error")}}
         "entered-in-error"
       {{else if (eq normalizedCode "nullified")}}
         "entered-in-error"
+      {{else if (eq normalizedCode "draft")}}
+        "draft"
       {{else if (eq normalizedCode "on-hold")}}
-        "on-hold"
+        "draft"
+      {{else if (eq normalizedCode "in-progress")}}
+        "draft"
+      {{else if (eq normalizedCode "active")}}
+        "draft"
```

Comment:
re: https://build.fhir.org/ig/HL7/ccda-on-fhir/branches/master/ConceptMap-CF-MedicationStatus.html

`active`

Reply:
There's no `draft`, here and other ones